src/client/swagger/unity.yml

```diff
@@ -420,6 +456,54 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
+  '/orgs/{orgId}/limits':
```

Comment:
I think this is already implemented here:

https://github.com/influxdata/ui/blob/master/src/cloud/components/GetAssetLimits.tsx

Where we get this https://github.com/influxdata/ui/blob/master/src/cloud/apis/limits.ts#L20

Which is defined here:

https://github.com/influxdata/ui/blob/master/src/types/cloud.ts#L1

Reply:
Assuming that the user has permissions to read those limits, it's probably fine to get them directly from IDPE. I thought we needed Quartz' operator token for that, but if not, then we're good there.

Reply:
Ya, I don't think there should be an issue getting it from IDPE. Especially since this implementation won't focus on integrating Cancelled users

Reply:
So this limits section will need to exist in the quartz api for the operator pages (not for the billing ones though). I think we can lock it down to be an operator. The main reason for this is  if I am an operator, i need to be able to view and update limits for any org no matter if my idpe user has access to it or not. For this reason, we will need to get these through quartz because the quartz token does have access to view these.